css/teams/1/landing-page.css

```diff
@@ -1,2 +1,336 @@
 @import url(../../colors.css);
+@import url(../../typography.css);
```

Comment:
you do not need to include it here, but yes in the html. Unless you are using variables from the typography file.